hathor/cli/quick_test.py

```diff
@@ -21,15 +21,10 @@ class QuickTest(RunNode):
 
     def register_resources(self, args: Namespace) -> None:
```

Comment:
Why did you change this method?

Reply:
I was testing the load times when not using all the indexes and when it finished loading adding a transaction doesn't work. But also the sync itself would also break, depending on what I removed. This changes shouldn't have been committed though, I'll remove them (although it could be somewhat useful if I added an option to change the behavior, I'd rather do it separately, and also optionally not enable the p2p, one of the tests I made caused an alert because of how the node interacted with other nodes on the p2p).

Reply:
Done.

Reply:
Thanks!